hr_employee_relative/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Employee Relatives",
+    "version": "13.0.1.0.0",
```

Comment:
Please split the isort and actual migration in two commits as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch

Reply:
it's my mistake. I think change version before pre-commit. thank you for your review :)